docker/otel-collector/config.yaml

```
@@ -104,7 +104,7 @@ exporters:
       max_interval: 30s
       max_elapsed_time: 300s
   clickhouse:
-    endpoint: tcp://${env:CLICKHOUSE_SERVER_ENDPOINT}?dial_timeout=10s&compress=lz4
+    endpoint: ${env:CLICKHOUSE_SERVER_ENDPOINT}
```

```suggestion
    endpoint: ${env:CLICKHOUSE_ENDPOINT}
```

if we were to follow the naming pattern on username/password below?

---

Good idea. Pushed the update